.github/workflows/contrib_rerun_py.yml

```diff
@@ -100,6 +100,11 @@ jobs:
         shell: bash
         run: cd rerun_py/tests && pytest -c ../pyproject.toml
 
+      # TODO(#5332): only run this nightly or on main
+      - name: Run Python tests on all Python versions
+        shell: bash
+        run: nox -s tests
```

Comment:
Are you sure nox is installed by default in the runner? I dont think it's in the `requirements-*.txt` as this should not depend on a venv (nox actually creates venv for each subtasks, like pixi).

Reply:
Also, yeah, this takes quite some time, so it might be really annoying to have on PRs

Reply:
this is the CI - of course I'm not sure

Reply:
I put this in the completely wrong file too

Reply:
I removed it again. We'll do it in https://github.com/rerun-io/rerun/issues/2445 instead, any year now